.github/workflows/release-container-image.yaml

```diff
@@ -0,0 +1,76 @@
+name: Release Container Image
+
+on:
+  schedule:
+    - cron: '3 3 * * *'
+  pull_request:
+  push:
+    branches: master
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  container:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
```

Comment:
Why is this action required?

Reply:
Ah, I guess now I see it. It's required to build it for multiple architectures. Could you please add a comment explaining that.

Reply:
I think Docker Buildx uses it. The job is how upstream documents we should use the action: https://github.com/docker/build-push-action#complete-workflow

Reply:
IMHO no comment needes, as it is how upstream documented it.